packages/suite/src/components/suite/modals/AddAccount/components/NetworkSelect.tsx

```diff
@@ -53,9 +53,10 @@ interface Props {
     network: Network;
     internalNetworks: Network[];
     setSelectedNetwork: (n: Network) => void;
+    pinNetwork?: boolean;
```

Comment:
i would prefer to name it `isDisabled`

Reply:
I kind of disagree as the `pinNetwork` conveys a "business" meaning that the user cannot do something, while `isDisabled` is only a lower level technical way how to achieve it. 
However, I will change it. Would you like me to rename it also in the `Wrapper` component?

Reply:
in this particular component it's used only for disabling Select component and i think it would be more intuitive to read `<NetworkSelect isDisabled={props.pinNetwork} />` in upper scope (Wrapper)